client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/patterns-data.ts

```diff
@@ -1,54 +1,76 @@
+import { translate } from 'i18n-calypso';
 import type { Pattern } from './types';
 
+const header = translate( 'Header' );
```

Comment:
It won't be translated to the correct language when this file is loaded as we might not have a user locale at that time.

Reply:
Mind=blown

Reply:
Nice catch 👍  I've updated it to return the patterns in a function, and now it works. What do you think about the implementation?
<img width="322" alt="Screen Shot 2565-12-02 at 12 12 54" src="https://user-images.githubusercontent.com/1881481/205219846-6ebfd1d7-7b44-4fc0-8c57-cb0c504c4c45.png">

Reply:
BTW There translations in Spanish need some work. I'll create an issue to check the translations in the main languages.